boards/arc/em_starterkit/arc_mpu_regions.c

```diff
@@ -50,11 +52,13 @@ static struct arc_mpu_region mpu_regions[] = {
 			AUX_MPU_RDP_KE | AUX_MPU_RDP_UE),
 #endif
 #endif /* ARC_MPU_VER == 3 */
+#ifndef CONFIG_ARC_SECURE_FIRMWARE
```

Comment:
ultimately, peripherals should be split over Normal and Secure mode. Maybe normal and secure board dts files can be used for that.

Reply:
It depends on the application. Currently, I just let the peripherals shared between secure and normal mode